app/assets/fonts/work-icons/work-icons.svg

```diff
@@ -3,14 +3,15 @@
 <svg xmlns="http://www.w3.org/2000/svg">
 <metadata>Generated by Fontastic.me</metadata>
 <defs>
-<font id="work-icons" horiz-adv-x="512">
-<font-face font-family="work-icons" units-per-em="512" ascent="480" descent="-32"/>
+<font id="untitled-font-2" horiz-adv-x="512">
+<font-face font-family="untitled-font-2" units-per-em="512" ascent="480" descent="-32"/>
```

Comment:
I'm fine with the new icon — but can we give it a real name?

Reply:
Yeah. That was an oversight.